crates/virtio-queue/src/state.rs

```diff
@@ -74,9 +74,16 @@ impl QueueState {
         val: u16,
         order: Ordering,
     ) -> Result<(), Error> {
+        // The multiplication can not overflow an u64 since we are multiplying an u16 with a
```

Comment:
Can we combine these two comments into one saying that the following operations cannot overflow because they're working with small numbers? I find the comments to be too much and getting in the way of the readability. Same for the other ones where there are 2 subsequent operations.

Reply:
Done, I also combined some of the operations.